generators/app/index.js

```diff
@@ -4,6 +4,18 @@ const defaults = require('./templates/defaults.json')
 const beautify = require('gulp-beautify')
 const filter = require('gulp-filter')
 const terminalLink = require('terminal-link')
+const selfsigned = require('selfsigned');
```

Comment:
this needs to be added to package.json and you need to commit a fresh package-lock